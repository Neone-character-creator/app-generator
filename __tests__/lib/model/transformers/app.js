const transformer = require("../../../../lib/model/transformers/app");

describe("app config transformer", () => {
    it("leaves an empty object unchanged", () => {
       const original = {};
       const transformed = transformer(original);
       expect(transformed).toEqual({});
    });
    it("replaces string shorthand with configuration object", () => {
        const original = {
            string: "string"
        };
        const transformed = transformer(original);
        expect(transformed).toEqual({
            string: {
                type: "string"
            }
        });
    })
});