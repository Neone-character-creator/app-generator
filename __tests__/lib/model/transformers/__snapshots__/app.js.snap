// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app config transformer throws an error if views undefined 1`] = `"views undefined"`;
